ruleset.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Roots">
```

Comment:
the ruleset is for all roots projects, not just sage — this will be added to our other repos